bitwarden_license/bit-web/src/app/secrets-manager/service-accounts/service-accounts-list.component.html

```diff
@@ -79,6 +79,12 @@
         ></button>
       </td>
       <bit-menu #serviceAccountMenu>
+        <button type="button" bitMenuItem (click)="editServiceAccountEvent.emit(serviceAccount.id)">
+          <i class="bwi bwi-fw bwi-pencil" aria-hidden="true"></i>
+          <span class="">
+            {{ "editServiceAccount" | i18n }}
+          </span>
+        </button>
```

Comment:
```suggestion
        <button type="button" bitMenuItem (click)="editServiceAccountEvent.emit(serviceAccount.id)">
          <i class="bwi bwi-fw bwi-pencil" aria-hidden="true"></i>
            {{ "editServiceAccount" | i18n }}
        </button>
```

I don't think we need the span here do we?